src/main/src/main/java/org/geoserver/catalog/impl/WMSLayerInfoImpl.java

```diff
@@ -71,12 +73,15 @@ public void reset() {
 
         try {
             // read from cap doc
-            return getWMSLayer(null)
-                    .getStyles()
-                    .stream()
-                    .map(s -> s.getName())
-                    .collect(Collectors.toList());
-
+            List<String> remoteStyles = new ArrayList<String>();
```

Comment:
This change seems unrelated, mixup with other work?

Reply:
will remove